config/database.php

```diff
@@ -11,48 +11,12 @@
 use Illuminate\Support\Str;
 
 return [
-
-    /*
-    |--------------------------------------------------------------------------
-    | Default Database Connection Name
-    |--------------------------------------------------------------------------
-    |
-    | Here you may specify which of the database connections below you wish
-    | to use as your default connection for all database work. Of course
-    | you may use many connections at once using the Database library.
-    |
-    */
-
     'default' => env('DB_CONNECTION', env('DBTEST') ? 'testing' : 'mysql'),
-    /*
-    |--------------------------------------------------------------------------
-    | Database Connections
-    |--------------------------------------------------------------------------
-    |
-    | Here are each of the database connections setup for your application.
-    | Of course, examples of configuring each database platform that is
-    | supported by Laravel is shown below to make development simple.
-    |
-    |
-    | All database work in Laravel is done through the PHP PDO facilities
-    | so make sure you have the driver for your particular database of
-    | choice installed on your machine before you begin development.
-    |
-    */
 
     'connections' => [
-
-        'sqlite' => [
```

Comment:
I'm assuming Shift has removed all of these config/* changes because they are unused (haven't checked them all). They still exist in Laravel 12 so I wonder if we should just keep them or if it's unnecessary clutter?

Reply:
this is from laravel upstream changes, its just built into the framework rather than "frontend" now.
For example:

https://github.com/laravel/framework/blob/7bf607bd6f9c919e08de96582170fe64a77037dc/config/database.php#L32-L36